src/udev/udevadm-trigger.c

```diff
@@ -266,6 +269,7 @@ int trigger_main(int argc, char *argv[], void *userdata) {
                 { "help",                  no_argument,       NULL, 'h'                       },
                 { "uuid",                  no_argument,       NULL, ARG_UUID                  },
                 { "prioritized-subsystem", required_argument, NULL, ARG_PRIORITIZED_SUBSYSTEM },
+                { "initialized",           required_argument, NULL, ARG_INITIALIZED           },
```

Comment:
so far all flags directly mapping to matches where called xyz-match and xyz-nomatch. That might not be ideal, but it's also not completely bad, so I'd stick to that?

Reply:
Originally @dakr introduced this as `--database-match` and `--database-nomatch`, but here existence of database is used as if the device is initialized (or processed by udevd), and its connection is not trivial for users, I think. So, I changed the name.
If you'd like to 'match' and 'nomatch', then, how about '--match-initialized' '--match-uninitialized'? Though, still I prefer the current form: '--initialized'.

Reply:
Hmm I think, `--initialized-match=` + `--initialized-nomatch=` sounds like the way to go, is it fits into both naming schemes we so far adopted. i.e. match/unmatch and "initialized"